src/Microsoft.VisualStudio.Threading.Tests/TestBase.cs

```diff
@@ -13,7 +13,7 @@ public abstract class TestBase
     {
         protected const int AsyncDelay = 500;
 
-        protected const int TestTimeout = 1000;
+        protected const int TestTimeout = 5000;
```

Comment:
:bulb: Consider using `TimeSpan` instead of `int`.

Reply:
IIRC This must be const, so a TimeSpan isn't an option.

Reply:
Just checked... it _could_ be `static readonly` but it's actually used in several places where being in milliseconds makes it much easier to multiply by various factors to get longer/shorter timeout values.